azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -351,6 +385,23 @@ azkaban.EditTableView = Backbone.View.extend({
     $(valueData).addClass("spanValue");
     $(valueData).text(value);
 
+    var idSelect = document.createElement("select");
+    idSelect.setAttribute("class", "form-control");
+
+    // global option (flowOverride)
+    idSelect.options[0] = new Option("", "");
+
+    // node-specific options (nodeOverride)
+    for (var i = 0; i < nodeIdsAndNestedIds.length; ++i) {
+      var idOrNestedId = nodeIdsAndNestedIds[i];
+      idSelect.options[i + 1] = new Option(idOrNestedId, idOrNestedId);
+      if (jobOrFlow == idOrNestedId) {
+        idSelect.options[i + 1].selected = true;
+      }
+    }
```

Comment:
Could you post a screenshot to show what exactly happened? I'm not 100% sure, sorry.

In general I can comment that on purpose using plain job names was allowed: that way one is able to add a single override that targets all instances of a job that is re-used inside multiple sub-flows – even if those sub-flows have different ids and hence the nested ids are different for the instances of the re-used job.

Reply:
Here is the flow I used:
<img width="497" alt="Screen Shot 2021-02-05 at 5 29 39 PM" src="https://user-images.githubusercontent.com/44478711/107104795-13ef9f80-67d8-11eb-8bef-072532e4fe08.png">
Here is the generated list of nodes:
<img width="897" alt="Screen Shot 2021-02-05 at 5 29 23 PM" src="https://user-images.githubusercontent.com/44478711/107104806-21a52500-67d8-11eb-98ab-dc698384f4a6.png">
Notice job `jobE1B` could refer to the one inside nested flow `emb1` or  inside`emb2`.

Pros-Cons of having 2 ways (`node id` and `nested node id`) of targeting the same node: 
-Can be confusing for users in terms of which one to use for overrides, or in case they have complex flows and specify both they could see properties resolved to unexpected values because of the precedence of `nested id` over the node name which is not a good experience IMO. 
-It's more complex to maintain.  
-It's only benefit seems to be the ability to override properties of jobs with the same name in one go but having jobs with the same name should be rare and I don't think the benefits outweigh the cons.

Reply:
In addition, the team at LinkedIn is working on revamping the APIs (will be published soon) and for the create execution endpoint to target nodes in parameters we're requiring to provide their fully qualified name to avoid ambiguities and unexpected behaviors. Therefore it makes sense to have the same requirement on this new feature.

Reply:
True, that is better design. I'll change to full nested id only.

As a side note, I suppose the work can be pushed down to custom client libraries if anyone needs an easy way to target a leaf node by name inside different sub-flows.

Reply:
Done in https://github.com/azkaban/azkaban/pull/2597/commits/647fa74f92e16effa772907c5b9673bf21608a4c – it seems that the help text in the UI doesn't need to be changed :)

Reply:
Thanks!